scripts/endToEndExtraction/remove-testcases.py

```diff
@@ -61,7 +61,7 @@ def get_checks(content, sol_file_path):
             constructors.append(line)
         if line.startswith("ABI_CHECK") or line.startswith("BOOST_REQUIRE"):
             checks.append(line)
-    with open(sol_file_path, "r") as sol_file:
+    with open(sol_file_path, "r", encoding='utf8') as sol_file:
```

Comment:
I could not test this and `verify-testcases.py` but the only thing we have to worry about here are test cases for broken unicode sequences. I think `SolidityEndToEndTest` does not have any of these and they're all in standalone files.

Reply:
What was the default encoding previously?

Reply:
It seems to use `locale.getpreferredencoding(False)`. Which is `utf-8` in my system.

Reply:
Yeah, see https://github.com/ethereum/solidity/pull/11832#issuecomment-903999899. it's system-dependent. Which is why pylint now warns about it :)

It's only a problem if you expect byte sequences that are invalid UTF-8 in that file.